app/package.json

```diff
@@ -64,6 +64,7 @@
     "next-compose-plugins": "^2.2.1",
     "next-connect": "^0.9.1",
     "next-optimized-images": "^2.6.2",
+    "next-plausible": "^2.0.0",
```

Comment:
integrated with NextJS:
https://github.com/4lejandrito/next-plausible

Reply:
we need to trust 4lejandrito...